Products/CMFPlone/PasswordResetTool.py

```diff
@@ -134,33 +137,26 @@ def resetPassword(self, userid, randomstring, password):
         # clean out the request
         del self._requests[randomstring]
         self._p_changed = 1
-    ## Implementation ##
-
-    # external
-
-    security.declareProtected(ManagePortal, 'setExpirationTimeout')
 
+    @security.protected(ManagePortal)
     def setExpirationTimeout(self, timedelta):
         """Set the length of time a reset request will be valid in days.
         """
         self._timedelta = abs(timedelta)
 
-    security.declarePublic('getExpirationTimeout')
-
+    @security.public
     def getExpirationTimeout(self):
         """Get the length of time a reset request will be valid.
         """
         return self._timedelta
 
-    security.declarePublic('checkUser')
-
+    @security.declarePublic
     def checkUser(self):
         """Returns a boolean representing the state of 'user check' as described
         in 'toggleUserCheck'. True means on, and is the default."""
         return self._user_check
 
-    security.declarePublic('verifyKey')
-
+    @security.public
```

Comment:
@hvelarde ``security.declarePublic`` and ``@security.public`` are the same with some extra bonus as not being able to make typos (what if you write security.declarePublic('fao') and as @jensens already said, it works on python 3 as well